code/modules/mob/living/basic/heretic/heretic_summon.dm

```diff
@@ -1,16 +1,14 @@
 /mob/living/basic/heretic_summon
```

Comment:
ehhh i don't like that we're constantly building off this subtype but

A) These guys probably shouldn't appear in any other context than being a "heretic summon"
B) it's already in

c'est la vie

Reply:
If you want me to kill it then say so now before it's too late.
It's only holding some default values (some of which needn't be there at all frankly).

Reply:
i would rather it be killed, the fact that you had to genericize some of the values in this PR alone is cause enough for concern.

Reply:
Personally I think it's a beneficial type, particularly for being able to do `subtypesof(/mob/living/basic/heretic_summon)`

Reply:
> Personally I think it's a beneficial type, particularly for being able to do `subtypesof(/mob/living/basic/heretic_summon)`

That's true, we are actually doing this in code and the alternative is to maintain a big global list.

Reply:
yeah that's sorta what i meant with the point A since we need them to build menus and whatever